src/Refinery/To/Transformation/NewMethodTransformation.php

```diff
@@ -17,38 +17,38 @@
 class NewMethodTransformation implements Transformation
 {
 	use DeriveApplyToFromTransform;
+
 	/**
 	 * @var
 	 */
-	private $className;
+	private $object;
 
 	/**
 	 * @var
 	 */
 	private $method;
 
 	/**
-	 * @param string $className
+	 * @param string $object
 	 * @param string $methodToCall
 	 */
-	public function __construct(string $className, string $methodToCall)
+	public function __construct($object, string $methodToCall)
 	{
-		if (false === class_exists($className)) {
+		if (false === is_object($object)) {
 			throw new ConstraintViolationException(
 				'The first parameter MUST be an object',
 				'first_parameter_must_be_an_object'
 			);
 		}
 
-
-		if (false === method_exists($className, $methodToCall)) {
+		if (false === method_exists($object, $methodToCall)) {
 			throw new ConstraintViolationException(
```

Comment:
Please use `\InvalidArgumentException` here. If the method does not exist on the object this does not mean that some input violated a constraint, but instead that some developer constructed a transformation improperly.